upup/pkg/fi/cloudup/gcetasks/storagebucketiam.go

```diff
@@ -147,10 +147,10 @@ func (_ *StorageBucketIam) RenderGCE(t *gce.GCEAPITarget, a, e, changes *Storage
 	return nil
 }
 
-type terraformStorageBucketIam struct {
```

Comment:
line 158 refers to this struct , but commented. So I comment this struct not remove it.